src/api-reference/common/list-item/v4.list-item-bulk.markdown

```diff
@@ -0,0 +1,494 @@
+* [RFC 7230 Content-Length](https://tools.ietf.org/html/rfc7230#section-3.3.2)
+* [RFC 7231 Content-Type](https://tools.ietf.org/html/rfc7231#section-3.1.1.5)
+* [RFC 7231 Date](https://tools.ietf.org/html/rfc7231#section-7.1.1.2)
+* [RFC 7234 Cache-Control](https://tools.ietf.org/html/rfc7234#section-5.2)
+
+
+#### Payload
+
+* [Bulk List Item Response](#schema-bulk-list-item-response)
+* [Bulk List Error](#schema-bulk-list-error)
+* [Bulk List Item Request](#schema-bulk-list-item-request)
+* [Error Message](#schema-error-message)
+
+### Example
+
+#### Request
+
+```http
+PATCH https://us.api.concursolutions.com/list/v4/lists/530ae443-3b22-a44c-8211-c66d96c58d9e/bulk
+Accept: application/json
+Accept-Language: en
+Content-Type: application/json
+Authorization: Bearer {token}
+```
+
+Example 1:
+```json
+{
+  "requests": [
+    {
+      "code": "ITEM",
+      "value": "ITEM UPDATED"
+    }
+  ]
+}
+```
+
+Example 2:
+```json
+{
+  "requests": [
+    {
+      "code": "ITEM",
+      "deleted": true
+    }
+  ]
+}
+```
+
+Example 3:
+```json
+{
+  "requests": [
+    {
+      "code": "ITEM",
+      "value": "ITEM"
+    },
+    {
+      "code": "ITEM_TWO",
+      "value": "ITEM_TWO UPDATED"
+    },
+    {
+      "code": "ITEM_THREE",
+      "value": "ITEM_THREE UPDATED"
+    }
+  ]
+}
+```
+
+#### Response
+
+```shell
+HTTP/1.1 200
+concur-correlationid: 46d1b015-635e-4bca-be76-2ed39c122065 
+content-length: 289
+content-type: application/json;charset=UTF-8
+location: https://us.api.concursolutions.com/list/v4/lists/530ae443-3b22-a44c-8211-c66d96c58d9e/bulk
+date: Fri,26 Aug 2022 17:11:35 GMT 
+cache-control: no-cache, private
+```
+
+Example 1:
+```json
+{
+  "status": "SUCCESS",
+  "recordsSucceeded": 1,
+  "recordsFailed": 0,
+  "errors": []
+}
+
+```
+
+Example 2:
+```json
+{
+  "status": "SUCCESS",
+  "recordsSucceeded": 1,
+  "recordsFailed": 0,
+  "errors": []
+}
+
+```
+
+Example 3:
+```json
+{
+  "status": "PARTIAL_SUCCESS",
+  "recordsSucceeded": 2,
+  "recordsFailed": 1,
+  "errors": [
+    {
+      "message": "The list item has been deleted",
+      "listItem": {
+        "code": "ITEM",
+        "value": "ITEM"
+      }
+    }
+  ]
+}
+```
+
+## <a name="schema"></a>Schema
+
+### <a name="schema-bulk-list-item-create-request"></a>Bulk List Item Create Request
+|Name|Type|Format|Description|
+|---|---|---|---|
+|`requests`|`list`|[`BulkListItemCreateRequestPart`](#schema-bulk-list-item-create-request-part)|The requests in the bulk list create request|
+
+
+### <a name="schema-bulk-list-item-create-request-part"></a>Bulk List Item Create Request Part
+|Name|Type|Format|Description|
+|---|---|---|---|
+|`shortCode`|`string`|-|List item short code|
+|`value`|`string`|-|List item value|
+|`parentCode`|`string`|-|The long code of parent list item|
```

Comment:
```suggestion
|`parentCode`|`string`|-|**Required** The long code of parent list item|
```

Reply:
Hm, I believe the long code of the parent list item is only required when creating list items under another list item in the same list. If we are creating first level list items, then the long code is not required. Do you think we should still mark the `parentCode` as `Required`?

Reply:
My mistake, only shortCode and value should be required. parentCode should not be required.

Reply:
Gotcha, thanks for clarifying! Added the `Required` to `shortCod` and `value`